src/renderer/components/CollabPicker/CollabPicker.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import './CollabPicker.scss';
+import TextField from '../TextField/TextField';
+import Button from '../Button/Button';
+import Select from '@mui/material/Select';
+import { MenuItem } from '@mui/material';
+import { useState } from 'react';
+
+interface Value {
+  username: string;
+  role: number;
+}
+
+interface Options {
+  value: Value;
+  label: string;
+}
+
+interface Props {
+  value?: Value;
+  options?: Options;
+  onChange?: (...args: any[]) => void;
+}
```

Comment:
The `Props` interface should be changed to account for the `Option` interface name change and the type of `onChange` needs to be properly set. An `onAdd` has to be added as well.

```ts
interface Props {
  value?: Value;
  options?: Option[];
  onChange?: (value: Value | undefined) => void;
  onAdd?: (value: Value) => void;
}
```